multimodal/setup.py

```diff
@@ -24,7 +24,7 @@
     "scipy",
     "pandas",
     "scikit-learn",
-    "Pillow",
+    "Pillow>=9.1.0",
```

Comment:
This will simply override the `_setup_utils` range: https://github.com/autogluon/autogluon/blob/master/core/src/autogluon/core/_setup_utils.py#L27

We should have a consistent range. I believe lower bound 9.3.0 was set due to a security issue with earlier versions.

Reply:
Okay, there is no bug then. The Colab issue can be resolved by restarting the runtime.